map/dynamic_object_removal/src/dynamic_object_removal.cpp

```diff
@@ -0,0 +1,88 @@
+#include "dynamic_object_removal.hpp"
+
+// Constructor
+DynamicObjectRemoval::DynamicObjectRemoval() : Node("dynamic_object_removal_node")
+{
+  pcl_publisher_ = this->create_publisher<sensor_msgs::msg::PointCloud2>("output/pointcloud", 10);
+
+  // Subscribe to topics
+  pcl_sub_.subscribe(this, "input/pointcloud");
```

Comment:
These two subscriber cause problem because of incompatible qos policy. You may update like this:

```
pcl_sub_.subscribe(this, "input/pointcloud", rmw_qos_profile_sensor_data);
obj_sub_.subscribe(this, "input/objects", rmw_qos_profile_sensor_data);
```